Berksfile

```diff
@@ -3,10 +3,8 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'apt', '~> 3.0.0'
-  cookbook 'yum', '~> 3.10.0'
-  cookbook 'zypper', '~> 0.2.1'
-  cookbook 'pacman', '~> 1.1.1'
```

Comment:
arch testing is removed since you can't install omnibus on arch.  zypper isn't actually required. zypper auto refreshes the repos when you install. all that cookbook does is a refresh.